src/main/java/com/dre/brewery/configuration/files/Config.java

```diff
@@ -165,6 +165,9 @@ public float getStumblePercent() {
     @LocalizedComment("config.agingYearDuration")
     private int agingYearDuration = 20;
 
+    @LocalizedComment("config.newBarrelTypeAlgorithm")
+    private boolean newBarrelTypeAlgorithm = false;
```

Comment:
Default false or true; what is best?

From my point of view, this should be true, as it adds to the immersion, and does not greatly affect gameplay